android/src/main/java/com/onesignal/rnonesignalandroid/RNOneSignal.java

```diff
@@ -165,6 +165,11 @@ private void removeObservers() {
     }
 
     private void removeHandlers() {
+        if(!oneSignalInitDone) {
```

Comment:
Why it is align with the rest code style?
Suggested change:
`if (!oneSignalInitDone) {`